### PR TITLE
S9: [flask] Update for Xen 4.12

### PR DESCRIPTION
**Pull request by tsirakisn**
- add get_cpu_policy
- remove set_max_evtchn and set_gnttab_limits
- grant dom0 access to resource_map of service VMs
- add argo definitions to match xen flask

OXT-1550
OXT-1563

Signed-off-by: Nicholas Tsirakis <tsirakisn@ainfosec.com>

### CODE REVIEW

File: policy/flask/access_vectors [hunk 1]
```diff
@@ -248,10 +246,10 @@ class domain2
     mem_sharing
 # XEN_DOMCTL_psr_alloc
     psr_alloc
-# XEN_DOMCTL_set_gnttab_limits
-    set_gnttab_limits
```
Comment (tsirakisn) [inline policy/flask/access_vectors:252]: set_gnttab_limits removed as per https://github.com/xen-project/xen/commit/ae8b8bc599ce2c1fc42f00a30d5e35a48c3e970c

File: policy/flask/access_vectors [hunk 2]
```diff
@@ -222,8 +222,6 @@ class domain2
     setscheduler
 # XENMEM_claim_pages
     setclaim
-# XEN_DOMCTL_set_max_evtchn
-    set_max_evtchn
```
Comment (tsirakisn) [inline policy/flask/access_vectors:226]: set_max_evtchn removed as per https://github.com/xen-project/xen/commit/4a83497635056d33fe20ef705f35617b1003a276

File: policy/flask/access_vectors [hunk 3]
```diff
@@ -248,10 +246,10 @@ class domain2
     mem_sharing
 # XEN_DOMCTL_psr_alloc
     psr_alloc
-# XEN_DOMCTL_set_gnttab_limits
-    set_gnttab_limits
 # XENMEM_resource_map
     resource_map
+# XEN_DOMCTL_get_cpu_policy
+    get_cpu_policy
```
Comment (tsirakisn) [inline policy/flask/access_vectors:252]: get_cpu_policy added as per https://github.com/xen-project/xen/commit/60529dfeca145a8ec00f5813a4c7179f0c1bfb97

**Comment by eric-ch** [comment]
Build [$6563](http://openxt-builder.ainfosec.com:8010/builders/openxt64/builds/6563)

**Comment by jean-edouard** [comment]
Built here: http://openxt-builder.ainfosec.com:8010/builders/openxt64/builds/6583

**Comment by jean-edouard** [comment]
Built here: http://openxt-builder.ainfosec.com:8010/builders/openxt64/builds/6593

**Comment by jean-edouard** [comment]
LGTM, merging soon.

**Pull request merged**